src/DummyTest.java

```diff
@@ -112,25 +112,55 @@ void testDefenseBiggerThanZero() {
         assertEquals(entity.getDefense() >= 0, true, "Wrong defense");
     }
 
+    @Test
+    @DisplayName("Test attack")
+    void testAttack() {
+        // Arrange
+        Entity entity1;
+        Entity entity2;
+
+        //Act
+        entity1 = new Entity(100, 50, 40, 2, "Bob", "Bob story");
+        entity2 = new Entity(90, 40, 30, 2, "Mike", "Mike story");
+
+        // Assert
+        assertEquals(entity1.getAttack() - entity2.getDefense(), 20, "Wrong attack");
+
+        assertEquals(entity2.getAttack() - entity1.getDefense(), 0, "Wrong attack");
+       
+    }
+
+    @Test
+    @DisplayName("Test attack with attack() function")
+    void testAttackWithAttackFunction() {
+        // Arrange
+        Entity entity1;
+        Entity entity2;
+
+        //Act
+        entity1 = new Entity(100, 50, 35, 2, "Bob", "Bob story");
+        entity2 = new Entity(90, 40, 30, 2, "Mike", "Mike story");
+
+        // Assert
+        entity1.attack(entity2);
+        assertEquals(entity2.getHealth(), 70, "Wrong attack");
+
+        entity2.attack(entity1);
+        assertEquals(entity1.getHealth(), 95, "Wrong attack");
+       
```

Comment:
Na, ezek tetszenek, sorrend megint nem jo, de amugy jo. Viszont na ilyenbol kellene nem 1 teszt hanem 20 majd :-) Most mintanak eleg.

Reply:
@hegyhati 

Javítottam az assertEquals(expected, actual) -t. Beleraktam 1  @ParameterizedTest -et is. Kérlek ellenőrízd le.